packages/components/src/reactions_row/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import styles from './index.module.scss';
+import { REACTION_SIZE_S, REACTIONS } from '../constants';
+import { useReactionData } from './hooks';
+import ReactionIconBubble from '../reaction_icon_bubble';
+
+const ReactionsRow = ({
+  withCounter = true,
+  reactions,
+  onClick,
+  className: passedClassName,
+  ...cleanProps
+}) => {
+  const { list: reactionsList, count } = useReactionData(reactions);
+
+  if (withCounter && count === 0) return null;
+
+  const renderReaction = (reaction) => (
+    <ReactionIconBubble
+      size={REACTION_SIZE_S}
+      colorize
```

Comment:
looking through - colorized should be the default. go for `simple` to disable those styles (flip the condition in clsx: `isColored: !simple`)